kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/ConflictChecker.java

```diff
@@ -141,6 +154,15 @@ public TransactionRebaseState(long latestVersion) {
         public long getLatestVersion() {
             return latestVersion;
         }
+
+        /**
+         * Return the latest inCommitTimestamp of the table.
```

Comment:
Actually, this doesn't necessarily have to be the ICT. If the winning transaction does not have ICT enabled and our transaction is enabling it, we want to ensure that our ICT is greater than the file modification timestamp of the winning transaction.

Reply:
Yes. I think I use file modification time when ICT is not available [here](https://github.com/delta-io/delta/blob/280357508d8e1c3eaf4a854bdc2bb24ca8f35b1c/kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/ConflictChecker.java#L258)?